src/source-pcap-file-helper.c

```diff
@@ -30,10 +30,13 @@
 #include "util-profiling.h"
 #include "source-pcap-file.h"
 #include "util-exception-policy.h"
+#include "conf-yaml-loader.h"
 
 extern uint32_t max_pending_packets;
 extern PcapFileGlobalVars pcap_g;
 
+SC_ATOMIC_DECLARE(uint64_t, g_pcap_file_alerts);
```

Comment:
Seems odd to have this as a global variable... Could not `PcapFileFileVars` hold this info ? (sorry if this was already discussed previously) or another structure dedicated to the pcap file being read...